### PR TITLE
Add new group functionality and display group profiles via dynamic paths

### PR DESCRIPTION
**Pull request by alternadiva**

### CODE REVIEW

File: components/Groups/GroupsForm.js [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+        description: groupDescription,
+        location: groupLocation,
+        avatar: cloudinary.secure_url,
+        members: [],
+        public: publicity,
+      },
+    ]);
+  }
+
+  //Borrow fishing related items from our group
```
Comment (sonianb) [inline components/Groups/GroupsForm.js:52]: Yes, please!

File: pages/groups/[id].js [hunk 2]
```diff
@@ -0,0 +1,72 @@
+import { useState, useEffect } from "react";
+import supabase from "../../utils/supabaseClient";
+
+function Group(props) {
+  const [groupAdmin, setGroupAdmin] = useState("");
+
+  useEffect(() => {
+    async function admin() {
```
Comment (sonianb) [inline pages/groups/[id].js:8]: Nice group admin functionality! 👍

**Review by AsmahanM — APPROVED** [review]

**Review by minju25kim — APPROVED** [review]

**Comment by sonianb** [comment]
[Create form so user can create a new group](https://trello.com/c/YT2c4d0U/4-create-form-so-user-can-create-a-new-group)

**Comment by sonianb** [comment]
[User who creates a group is the admin of the group.](https://trello.com/c/LkEyMhzS/5-user-who-creates-a-group-is-the-admin-of-the-group)

**Comment by sonianb** [comment]
[User who creates a group is the admin of the group.](https://trello.com/c/LkEyMhzS/5-user-who-creates-a-group-is-the-admin-of-the-group)

**Pull request merged**